src/scenes/CreateEvent/components/CreateEventOutcomes/index.js

```diff
@@ -55,25 +55,26 @@ export default class CreateEventOutcomes extends Component {
 
   validate = (value, allValues) => {
     const { intl } = this.props;
+    const outcome = value || '';
```

Comment:
why don't we do `(value || '').toLowerCase()` since we use the `.toLowerCase()` a couple times below